.bashrc

```diff
@@ -4,6 +4,10 @@
 
 [[ -f /etc/bash_completion ]] && source /etc/bash_completion
 
+for file in ~/Dropbox/code/common-files/bash_functions/*.sh; do
```

Comment:
This path is not a good path for me.

Reply:
Same here. I think the path could be `~/bash_functions/*.sh` which should be symlinked because of stow. Though, I think we probably want to move that folder to `~/.common_files/bash_functions` or actually I think `~/.common_files/bin` and adding that to the PATH is probably a more common pattern in dotfiles.